app_dart/lib/src/service/bigquery.dart

```diff
@@ -22,4 +42,59 @@ class BigqueryService {
     );
     return BigqueryApi(client).tabledata;
   }
+
+  /// Return a [JobsResourceApi] with an authenticated [client]
+  Future<JobsResourceApi> defaultJobs() async {
+    final Client client = await accessClientProvider.createAccessClient(
+      scopes: const <String>[BigqueryApi.BigqueryScope],
+    );
+    return BigqueryApi(client).jobs;
+  }
+
+  /// Return the list of current builder statistic;
```

Comment:
Nit: doc for the table schema will help understand the logic.

Reply:
Which table schema are you referring to? `flutter-dashboard.datasite.luci_prod_build_status` is the table the sql query from, or do you mean the schema of the sql result? Either way, These schemas are abstracted away because I return a dart class instead.

Reply:
Yeah, I meant the table `luci_prod_build_status` result. From the code here, it is not clear what f[0]-f[6] are referring to. Since the table is located in bigquery (internally?), it may be useful to list the basic schema here.